gapis/resolve/framebuffer_changes.go

```diff
@@ -62,26 +63,23 @@ func (r *FramebufferChangesResolvable) Resolve(ctx context.Context) (interface{}
 		attachments: make([]framebufferAttachmentChanges, gfxapi.FramebufferAttachment_Color3+1),
 	}
 
-	s := c.NewState()
-	for i, a := range c.Atoms {
-		id = atom.ID(i)
-		if err := a.Mutate(ctx, s, nil); err != nil && err == context.Canceled {
-			return nil, err
-		}
+	sync.MutateWithSubcommands(ctx, r.Capture, atom.List{c.Atoms}, func(s *gfxapi.State, subcommandIndex []uint64, a atom.Atom) {
```

Comment:
I think it is worth of adding some comments to say, now the framebuffer_changes are collected through executing the callback along with the mutation of the atoms.